daprdocs/content/en/reference/components-reference/supported-bindings/rabbitmq.md

```diff
@@ -27,7 +27,7 @@ spec:
     value: "amqp://[username][:password]@host.domain[:port]"
   - name: durable
     value: "true"
-  - name: deleteWhenUnused
+  - name: deletedWhenUnused
```

Comment:
Thanks for this PR! However, `deletedWhenUnused` is the correct name for RabbitMQ pub/sub, while for the binding its actually `deleteWhenUnused`. Ideally these should be consistent, but they are not today...

Reply:
I might be misunderstanding your comment but I just tested using both `deleteWhenUnused` set to both `true` and `false`, as well as `deletedWhenUnused` and only the latter works. As in it results in queues being created with `auto delete` off.

https://github.com/search?q=org%3Adapr%20deletedWhen&type=code

Reply:
You used `deletedWhenUnused` with the *pub/sub* component, but this PR is changing the *binding* component. The binding component metadata is `deleteWhenUnused`.